server/sockets.ts

```diff
@@ -495,6 +504,11 @@ export class ServerStream extends Streams.ObjectReadWriteStream<string> {
 				}
 			}
 		}
+		if (Config.validateSocket && !Config.validateSocket.call(this, socket, socketip)) {
```

Comment:
```suggestion
		if (Config.validateSocket && Config.validateSocket.call(this, socket, socketip) === false) {
```

Probably `undefined` should not force a socket to be deleted.